Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -29,6 +29,10 @@ namespace Multiplayer
 {
     using namespace AzNetworking;
 
+    AZ_CVAR(int, editorsv_servertype, 1, nullptr, AZ::ConsoleFunctorFlags::DontReplicate,
```

Comment:
You can make AZ_CVAR enums, so instead of an abstract 0, 1, 2, or even a true/false had you used a boolean, you can support values like DedicatedServer or ClientServer.. see the protocol type inside the multiplayer system component if you need an example, where we allow customers to swap between TCP or UDP for all multiplayer traffic.